examples/without-map/app.js

```diff
@@ -1,7 +1,7 @@
 /* global window,document,fetch */
 import React, {Component} from 'react';
 import {render} from 'react-dom';
-import {MapController} from 'deck.gl/dist/controllers';
+import {MapController} from 'deck.gl';
```

Comment:
Should we be updating the examples at this stage? This app will not work with `npm start`. 
We also discussed about not depending on alpha packages in examples. @shaojingli